tools/tools.go

```diff
@@ -0,0 +1,14 @@
+//go:build tools
+// +build tools
```

Comment:
```suggestion
```

Ah, only one is necessary. Looks like a tool added the newer version of the build tag.